pages/event/[eventSlug]/ticket/index.vue

```
@@ -156,6 +156,35 @@
+          Manter agendamento
+        </ElButton>
+        <ElButton 
+          class="cancel-button"
+          :loading="state.loading"
+          @click="confirmCancel"
+        >
+          Cancelar
+        </ElButton>
+      </div>
+    </template>
+  </ElDialog>
```

ESLint

[error] The template root requires exactly one element.

(vue/no-multiple-template-root)